src/lib/bootloader/grub2_widgets.rb

```diff
@@ -299,15 +299,17 @@ def label
     def help
       # TRANSLATORS: TrustedGRUB2 is a name, don't translate it
       res = _("<p><b>Trusted Boot</b> will install TrustedGRUB2\n" \
-          "instead of regular GRUB2.</p>\n" \
-          "<p>It means measuring the integrity of the boot process,\n" \
+          "instead of regular GRUB2." \
+          "It means measuring the integrity of the boot process,\n" \
           "with the help from the hardware (a TPM, Trusted Platform Module,\n" \
-          "chip).</p>\n")
+          "chip).\n")
       if grub2.name == "grub2"
-        res += _("<p>First you need to make sure Trusted Boot is enabled in the BIOS\n" \
-          "setup (the setting may be named Security Chip, for example).</p>\n")
+        res += _(" First you need to make sure Trusted Boot is enabled in the BIOS\n" \
```

Comment:
The previous string always ends with a newline, so no initial space is needed.
```suggestion
        res += _("First you need to make sure Trusted Boot is enabled in the BIOS\n" \
```

Reply:
Ah, good point!